### ISSUE TITLE
EditContext: If elements attached by `EditContext`, what happens?

### ISSUE THREAD

**Issue by masayuki-nakano** [w3c/editing#346]
If there are 2 or more elements attached by `EditContext`, one can be a descendent of another. In my understanding, only most ancestor one can have focus, so `EditContext` of descendents do not work?

**Comment by alexkeng** [comment]
That's correct. EditContext is activated (which means getting user text input) only when the associated element is focused, so in your case the EditContext of descendants will never be activated.
